onnx_tf/backend.py

```diff
@@ -484,9 +487,11 @@ def _conv(cls, node, input_dict, transpose=False):
       warnings.warn("Unsupported kernel_shape attribute by Tensorflow in "
                     "Conv operator. The attribute will be ignored.",
                     UserWarning)
-    return [tf.nn.convolution(x, weights, "SAME", strides=strides,
+    tf_pads = cls.pad2tfpad(node.attrs["pads"])
+    output = tf.nn.bias_add(tf.nn.convolution(x, weights, tf_pads, strides=strides,
                               dilation_rate=dilations,
-                              data_format=data_format)]
+                              data_format=data_format), input_dict[node.inputs[2]], data_format="NCHW")
```

Comment:
Can you document the third parameter?  Also, have bias = input_dict[node.inputs[2]] at the top and use 'bias' in the call to tf.nn.bias_add().

Reply:
This documentation is coming in the latest PR.